src/test/helpers.test.ts

```diff
@@ -71,14 +71,16 @@ describe('test encodeUrlParams', () => {
 });
 
 describe('test decodedUrlParams', () => {
-  it('should decode when its URLSearchParams', () => {
-    expect(decodedUrlParams('')).toEqual({});
+  it('should decode the "params" query param from the URL', () => {
```

Comment:
I found the wording in these two tests a bit confusing.

Also I think the implementation of the two tests were swapped.